Projects/MoveFlow/Sources/View/MovingFlowHouseView.swift

```diff
@@ -5,33 +5,23 @@ import hCoreUI
 import hGraphQL
 
 struct MovingFlowHouseView: View {
-    @ObservedObject var vm: HouseInformationInputModel
+    @StateObject var houseInformationInputvm = HouseInformationInputModel()
```

Comment:
Can we have this as an observedObject? Now it gets cleaned if we go back to to address view.